include/cppkafka/kafka_handle_base.h

```diff
@@ -279,21 +303,17 @@ class CPPKAFKA_API KafkaHandleBase {
 private:
     static const std::chrono::milliseconds DEFAULT_TIMEOUT;
 
+    // It seems that destroy_flags maybe RD_KAFKA_DESTROY_F_NO_CONSUMER_CLOSE by default.
```

Comment:
I would say that the most appropriate place for such comments (i.e. suggestions, thoughts) is commit description :)